custom/auth/ante/ante.go

```diff
@@ -13,15 +14,17 @@ import (
 
 // HandlerOptions are the options required for constructing a default SDK AnteHandler.
 type HandlerOptions struct {
-	AccountKeeper      cosmosante.AccountKeeper
-	BankKeeper         BankKeeper
-	FeegrantKeeper     cosmosante.FeegrantKeeper
-	OracleKeeper       OracleKeeper
-	TreasuryKeeper     TreasuryKeeper
-	SignModeHandler    signing.SignModeHandler
-	SigGasConsumer     cosmosante.SignatureVerificationGasConsumer
-	IBCChannelKeeper   channelkeeper.Keeper
-	DistributionKeeper distributionkeeper.Keeper
+	AccountKeeper          cosmosante.AccountKeeper
+	BankKeeper             BankKeeper
+	FeegrantKeeper         cosmosante.FeegrantKeeper
+	OracleKeeper           OracleKeeper
+	TreasuryKeeper         TreasuryKeeper
+	SignModeHandler        signing.SignModeHandler
+	SigGasConsumer         cosmosante.SignatureVerificationGasConsumer
+	IBCChannelKeeper       channelkeeper.Keeper
+	DistributionKeeper     distributionkeeper.Keeper
+	FeeShareKeeper         feehshareante.FeeShareKeeper
+	FeeShareBankKeeperFork feehshareante.BankKeeper
```

Comment:
I am not super keen on having two different interfaces for the same keeper in the antehandler. Would it not be cleaner to simply add the methods you need the existing interface in custom/auth/ante/expected_keeper.go and have one less HandlerOptions to keep track of? :)

Reply:
I agree, will do. I figured from your other comment you wanted to keep this the same as before. Will fix later today